packages/central-server/src/apiV2/export/constructExportEmail.js

```diff
@@ -25,7 +30,23 @@ ${error}`;
 Note that you need to be logged in to the admin panel for it to work, and after clicking it once, you won't be able to download the file again.`;
 };
 
-export const constructExportEmail = responseBody => ({
-  subject: 'Your export from Tupaia',
-  message: constructMessage(responseBody),
-});
+const generateAttachments = async (responseBody, emailAsAttachment) => {
+  const { filePath } = responseBody;
+  if (emailAsAttachment) {
+    const fileName = path.basename(filePath);
+    const buffer = await fs.readFileSync(filePath);
+    return [{ filename: fileName, content: buffer }];
+  }
+  return [];
+};
+
+export const constructExportEmail = async (responseBody, req) => {
+  const { emailAsAttachment } = req.query;
```

Comment:
```suggestion
  const { emailExportFileMode } = req.query;
```
Where `emailExportFileMode: 'attachment' | 'downloadLink'` (and defaulting to `downloadLink`). Feels a little clearer to me what this configuration does and what the options are